internal/generator/vector/output/http/http.go

```diff
@@ -32,6 +33,11 @@ type = "http"
 inputs = {{.Inputs}}
 uri = "{{.URI}}"
 method = "{{.Method}}"
+{{with .Proxy -}}
+proxy.enabled = true
+proxy.http = {{.}}
```

Comment:
@alanconway is this needs to change as ?
```
proxy.http =  "{{.Proxy}}"
```

Reply:
fixed